pkg/cache/clusterqueue.go

```diff
@@ -82,13 +82,14 @@ type ClusterQueue struct {
 
 	AdmittedUsage FlavorResourceQuantities
 	// localQueues by (namespace/name).
-	localQueues                                map[string]*queue
-	podsReadyTracking                          bool
-	hasMissingFlavors                          bool
-	hasMissingOrInactiveAdmissionChecks        bool
-	hasMultipleSingleInstanceControllersChecks bool
-	admittedWorkloadsCount                     int
-	isStopped                                  bool
+	localQueues                                            map[string]*queue
+	podsReadyTracking                                      bool
+	hasMissingFlavors                                      bool
+	hasMissingOrInactiveAdmissionChecks                    bool
+	hasMultipleSingleInstanceControllersChecks             bool
+	hasFlavorIndependentAdmissionCheckAppliedAtFlavorLevel bool
```

Comment:
```suggestion
	hasFlavorIndependentAdmissionCheckAppliedPerFlavor bool
```
nit, shorter and for consistency